docs/fundamentals/dashboard/standalone-for-python.md

```diff
@@ -0,0 +1,197 @@
+
+> [!div class="checklist"]
+>
+> - Start the Aspire dashboard in standalone mode.
+> - Use the Aspire dashboard with a Python application.
+
+## Prerequisites
+
+To complete this tutorial, you need the following:
+
+- [Docker](https://docs.docker.com/get-docker/)
+  - You can use an alternative container runtime, but the commands in this article are for Docker.
+- [Python 3.12 or higher](https://www.python.org/downloads/) locally installed
+- A sample allication
+
+## Sample application
+
+This quickstart can be completed using either Flask, Django, or FastAPI. A sample application in each framework is provided to help you follow along with this quickstart. Download or clone the sample application to your local workstation.
+
+### [Flask](#tab/flask)
+
+```Console
+git clone https://github.com/Azure-Samples/msdocs-python-flask-webapp-quickstart
+```
+
+### [FastAPI](#tab/fastapi)
+
+```Console
+git clone https://github.com/Azure-Samples/msdocs-python-fastapi-webapp-quickstart.git
+```
+
+---
+
+To run the application locally:
+
+### [Flask](#tab/flask)
+
+1. Go to the application folder:
+
+    ```Console
+    cd msdocs-python-flask-webapp-quickstart
+    ```
+
+1. Create a virtual environment for the app:
+
+    [!INCLUDE [Virtual environment setup](./includes/standalone/virtual-environment-setup.md)]
+
+1. Install the dependencies:
+
+    ```Console
+    pip install -r requirements.txt
+    ```
+
+1. Run the app:
+
+    ```Console
+    flask run
+    ```
+
+1. Browse to the sample application at `http://localhost:5000` in a web browser.
+
+    :::image type="content" source="./media/standalone/run-flask-app-localhost.png" alt-text="Screenshot of the Flask app running locally in a browser":::
+
+### [FastAPI](#tab/fastapi)
+
+1. Go to the application folder:
+
+    ```Console
+    cd msdocs-python-fastapi-webapp-quickstart
+    ```
+
+1. Create a virtual environment for the app:
+
+    [!INCLUDE [Virtual environment setup](./includes/standalone/virtual-environment-setup.md)]
+
+1. Install the dependencies:
+
+    ```Console
+    pip install -r requirements.txt
+    ```
+
+1. Run the app:
+
+    ```Console
+    uvicorn main:app --reload
+    ```
+
+1. Browse to the sample application at `http://localhost:8000` in a web browser.
+
+    :::image type="content" source="./media/standalone/run-fastapi-app-localhost.png" alt-text="Screenshot of the FastAPI app running locally in a browser.":::
+
+---
+
+## Adding OpenTelemetry
+
+To use the Aspire dashboard with your Python application, you need to install the OpenTelemetry SDK and exporter. The OpenTelemetry SDK provides the API for instrumenting your application, and the exporter sends telemetry data to the Aspire dashboard.
+
+1. Install the OpenTelemetry SDK and exporter:
+
+    ```Console
+    pip install opentelemetry-api opentelemetry-sdk opentelemetry-exporter-otlp-proto-grpc
+    ```
+
```

Comment:
For FastAPI:

opentelemetry-instrumentation-fastapi

```python
from opentelemetry.instrumentation.fastapi import FastAPIInstrumentor

# Add this line after the app is created and OTEL is setup
FastAPIInstrumentor.instrument_app(app)
```


For Flask:

opentelemetry-instrumentation-flask

```python
from opentelemetry.instrumentation.flask import FlaskInstrumentor

# add this line after configure_grpc_otel() call
FlaskInstrumentor().instrument()
```

Reply:
I'll add a pivot that adds the framework specific instrumentation code